src/sass/_shame.scss

```diff
@@ -69,18 +69,6 @@ body .row {
   }
 }
 
-.review {
```

Comment:
Good question. I can't find it in any of the static pages. It looks like it was written by Courtney during the HCA buildout. (Not that it couldn't have been reused.)